docs/sources/administration/enterprise-licensing/_index.md

```diff
@@ -196,6 +196,12 @@ The active users limit is turned off immediately.
 
 Settings updates at runtime are not affected by an expired license.
 
+#### Email sharing
+
+Email-shared dashboards won't be available for external users anymore. These dashboards will become private again but you will be able to make them public to everybody if you want to.
+
+Configurations will be kept and restored if you update your license.
+
```

Comment:
I've tried to rewrite the documentation to use present tense and in active voice to be more consistent with the Grafana Labs style guide in https://grafana.com/docs/writers-toolkit/.

Let me know what you think, or use feel free to apply the suggestion if it is technically accurate and still makes sense.

```suggestion
#### Email sharing

External users can't access dashboards shared via email.
These dashboards are now private but you can make them public and accessible to everyone if you want to.

Grafana keeps your email configurations and restores them after you update your license.

```

Reply:
```suggestion
#### Email sharing

External users can't access dashboards shared via email anymore.
These dashboards are now private but you can make them public and accessible to everyone if you want to.

Grafana keeps your sharing configurations and restores them after you update your license.

```

@jdbaldry What about this? I'm afraid "email configurations" can be confusing.

Reply:
Yeah great proposal! I think you are right that "email configurations" is a bit confusing, "sharing configurations" is much better :)